### ISSUE TITLE
Support for other liquid experience

### ISSUE THREAD

**Issue by RetroJect** [DragonsPlusMinecraft/CreateEnchantmentIndustry-Fabric-Legacy#20]
### Describe the your request

I've been playing the [Versatile modpack](https://modrinth.com/modpack/versatile) which includes both Create Enchantment Industry and [XP Obelisk](https://www.curseforge.com/minecraft/mc-mods/xp-obelisk). Both mods add in liquid experience (although it seems only the XP obelisk version shows up in mods like JEI). The printer and disenchanter blocks will not accept the XP Obelisk version of liquid experience.

Is it possible to support other additions of liquid experience from other mods?

**Comment by MGJerry** [comment]
Agree, I hate having to wait around as the middle man to transfer all the XP in my backpack (from [Sophisticated Backpack (Fabric Port)](https://www.curseforge.com/minecraft/mc-mods/sophisticated-storage-unofficial-fabric-port) because we don't have the Experience Rotor like the Forge counterpart.
